internal/telemetry/event.go

```diff
@@ -201,16 +202,17 @@ func withOS() EventOpt {
 type Authenticator interface {
 	PublicAPIKey() string
 	PrivateAPIKey() string
+	AccessToken() string
 }
 
 func withAuthMethod(c Authenticator) EventOpt {
```

Comment:
are we defaulting to auth_method = nil if unauth or do we set this value to `unauth` somewhere else?

Reply:
yes, default to not set which seemed to be the common pattern rather than setting to unauth